pages/products/index.tsx

```diff
@@ -45,15 +65,28 @@ const Products = () => {
 
     return (
         <Panel>
-            <StatefulTable
+            <Table
                 columns={[
-                    { header: 'Product name', hash: 'name', render: ({ id, name }) => renderName(id, name), sortKey: 'name' },
-                    { header: 'Stock', hash: 'stock', render: ({ stock }) => renderStock(stock), sortKey: 'stock' },
-                    { header: 'Price', hash: 'price', render: ({ price }) => renderPrice(price), sortKey: 'price' },
-                    { header: 'Action', hideHeader: true, hash: 'id', render: ({ id }) => renderAction(id), sortKey: 'id' },
+                    { header: 'Product name', hash: 'name', render: ({ id, name }) => renderName(id, name), isSortable: true },
+                    { header: 'Stock', hash: 'stock', render: ({ stock }) => renderStock(stock), isSortable: true },
+                    { header: 'Price', hash: 'price', render: ({ price }) => renderPrice(price), isSortable: true },
+                    { header: 'Action', hideHeader: true, hash: 'id', render: ({ id }) => renderAction(id), isSortable: true },
```

Comment:
let's get rid of `isSortable` for the action column... there's this weird effect whereby the current implementation is causing the action column to be sortable by default and it shows a sort arrow for the column:
<img width="108" alt="Screen Shot 2021-09-24 at 12 09 08 PM" src="https://user-images.githubusercontent.com/38708175/134728252-12160087-b3d2-4e9d-8d23-ebbc66d9fc68.png">
Since the column contains only a 3 dot menu (`...`), sorting a column that shows same exact content seems odd.